mbbsd/email_challenge.cc

```diff
@@ -3,6 +3,12 @@ extern "C" {
 #include "daemons.h"
 }
 
+#ifdef USE_2FA
```

Comment:
Call this `USE_EMAIL_2FA` or something, in case later we have other forms of 2FA.